### DISCUSSION TITLE
Change root context from / to /abcdefg

### DISCUSSION THREAD

**Discussion by pretchu** [structurizr/onpremises#83] [Q&A] [answered]
### Question

Hello everybody, 

I need to setup structurizr with a custom root context, I changed the **structurizr.url** to reflect my configuration to **HTTP://my-structurizr.com/abcdefg** but I am receiving many 404 with static resources and the pages are not loading. Does anyone know how to do this?

Structurizr configurations
![image](https://github.com/structurizr/onpremises/assets/15239376/86b5be32-5c4d-4c31-8cb7-f32e8a4d76e8)

Error messages
![image](https://github.com/structurizr/onpremises/assets/15239376/852a296b-3214-4e8a-9c4c-efdb284a448e)
![image](https://github.com/structurizr/onpremises/assets/15239376/65fb71bd-cebd-4024-947c-153a2fd36cc7)

**Comment by simonbrowndotje** [comment, marked as answer]
As it says in the docs:

> the on-premises installation must be installed as the root web application, and is not designed to work otherwise

Sorry!

**Reply by pretchu** [reply]
Thank you for the answer ;)
